docs/node/daemon/config/settings.mdx

```diff
@@ -19,33 +19,47 @@ This section provides a detailed overview of the operational parameters availabl
+   - `pg_db_port`: PostgreSQL database port (may be omitted for UNIX socket hosts). The default is `5432`.
 
-- `pg_db_user`: PostgreSQL database user (should be a "superuser"). The default is `kwild`.
+   - `pg_db_user`: PostgreSQL database user (should be a "superuser"). The default is `kwild`.
 
-- `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations).
+   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations).
 
-- `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
+   - `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
 
-- `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
+   - `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
 
-- `extension_endpoints`: This variable specifies the endpoints for extensions, separated by commas.
+   - `extension_endpoints`: This variable specifies the endpoints for extensions, separated by commas.
 
-- `tls_cert_file`: This variable specifies the file path containing a certificate to enable TLS on the gRPC server. This path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
+   - `tls_cert_file`: This variable specifies the file path containing a certificate to enable TLS on the gRPC server. This path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
 
-- `tls_key_file`: This variable specifies the file path containing the corresponding private key that enables TLS on the gRPC server. Like the `tls_cert_file`, this path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
+   - `tls_key_file`: This variable specifies the file path containing the corresponding private key that enables TLS on the gRPC server. Like the `tls_cert_file`, this path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
 
-- `hostname`: This variable sets the Kwil gRPC server's hostname.
+   - `hostname`: This variable sets the Kwil gRPC server's hostname.
+
+   - `snapshot_file`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start.
+
+   ### [app.snapshots]
+
+      This section contains configurations for managing the snapshots of the kwild database.
+
+      - `enabled`: This variable toggles the snapshot feature on or off. By default, the snapshot feature is disabled.
```

Comment:
Could be more descriptive of the feature itself here.  "This enables creation and serving of snapshots to support state sync on nodes joining the network." or similar

Reply:
I also wish we didn't begin each one with "This variable", but we'd have to change them all, so can leave that sentence structure.